src/Models/Constants/Interfaces.ts

```diff
@@ -141,10 +142,26 @@ export interface IConsumeCompositeCardProps extends ICardBaseProps {
 }
 
 export interface IAuthService {
-    login: () => void;
-    getToken: (
-        tokenFor?: 'azureManagement' | 'adx' | 'storage' | 'powerBI'
-    ) => Promise<string>;
+    /** the unique object id of the logged in user */
+    userObjectId?: string;
+    setActiveAccount: (account: any) => void;
+    login: (continuation?: (params: any) => void) => void;
```

Comment:
should the continuation function take the IAuthParams object? or do we really not know the type?

Reply:
no, continuation is a optional callback method after login is successful with purely customizable in terms of parameters. For example, in adt app we pass username, name, tenants and accounts as params to show the user name in user menu and pick up the certain account with a certain tenant id which is read from local storage or url. So it should be any in this case.